flink-streaming-scala/src/main/scala/org/apache/flink/streaming/api/scala/SplitStream.scala

```diff
@@ -29,6 +29,7 @@ import org.apache.flink.streaming.api.datastream.{ SplitStream => SplitJavaStrea
  * the appropriate method on this stream.
  */
 @Public
+@deprecated
```

Comment:
Could we add a deprecation message explaining why it shouldn't be used anymore? `@deprecated("reason", "deprecated since.")`